staging/src/k8s.io/apiserver/pkg/endpoints/metrics/metrics.go

```diff
@@ -109,6 +109,19 @@ var (
 		},
 		[]string{"verb", "dry_run", "group", "version", "resource", "subresource", "scope", "component"},
 	)
+	requestSloLatencies = compbasemetrics.NewHistogramVec(
+		&compbasemetrics.HistogramOpts{
+			Name: "apiserver_request_slo_duration_seconds",
```

Comment:
@logicalhan @dgrisonnet - any better naming suggestion here maybe?

Reply:
sadly, not really...

Reply:
I don't have anything either. I feel like even though "slo" is a bit weird, it will be hard to come up with something more explicit and easily differentiable from `apiserver_request_duration_seconds` than that. The current naming is also a good way to discourage users from using this metric for anything else than SLOs so I kinda like it this way.

Reply:
Two ideas but they're not much better imo.
`apiserver_request_processing_duration_seconds`
or
`apiserver_internal_request_duration_seconds`

Reply:
Thank you both. I think from the above I personally prefer the current one and I like the argument from @dgrisonnet - so let's maybe stick to what we have currently.